src/main/resources/db/migration/changes/UCSBDiningCommonsMenuItems.json

```
@@ -0,0 +1,60 @@
+{ "databaseChangeLog": [
+    {
+        "changeSet": {
+          "id": "UCSBDiningCommonsMenuItems-2",
+          "author": "MattP",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "UCSBDININGCOMMONSMENUITEMS"
+                  }
+                }
+              ]
+            }
+          ],
+          "changes": [
+            {
+              "createTable": {
+                "columns": [
+                {   
+                    "column": {
+                      "autoIncrement": true,
+                      "constraints": {
+                        "primaryKey": true,
+                        "primaryKeyName": "CONSTRAINT_3"
+                      },  
+                      "name": "ID",
+                      "type": "BIGINT"
+                    }   
+                  },
+                  {
+                    "column": {    
+                      "name": "CODE",
+                      "type": "VARCHAR(255)"
+                    }
+                  },
+                  {
+                    "column": {   
+                      "name": "LOCATION",
```

```suggestion
                      "name": "STATION",
```
 match var name from DCM.java entity